### ISSUE TITLE
How to improve the mask obtained by domain adaptation on bitmoji dataset

### ISSUE THREAD

**Issue by Riqzz** [seasonSH/SemanticStyleGAN#13]
I made fine-tuning on the *Bitmoji* dataset according to the *Domain Adaptation* section, and the segmentation mask obtained is still far from the truth. Could you please advise how to get results like the pre-trained model `BitMoji-512x512.pt`?

**Comment by seasonSH** [comment]
Have you tried to remove the --freeze_local flag for fine-tuning on BitMoji?

**Comment by Riqzz** [comment]
Yes. I found that this flag has a large impact on the quality of the generated image and the number of fine-tuning steps. Keeping it the model can get a very good mask. And if it is removed, the image can be generated very well, but the mask is much worse.

**Comment by seasonSH** [comment]
Yes. You have to limit the number of fine-tuning steps if you remove this flag.
